.github/workflows/submitExtension.yml

```diff
@@ -0,0 +1,72 @@
+name: Submit extension to browser store
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Bump Version
+        default: v1.0.0
+        required: true
+jobs:
+  bump-version:
+    name: Bump Package Version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update package.json version
+        uses: jossef/action-set-json-field@v1
+        with:
+          file: ./extension/package.json
+          field: version
+          value: ${{ github.event.inputs.version }}
+      - name: Update manifest.json version_name
+        uses: jossef/action-set-json-field@v1
+        with:
+          file: ./extension/public/manifest.json
+          field: version_name
+          value: ${{ github.event.inputs.version }}
+      - name: Get manifest.json version
+        id: manifest_version
+        uses: frabert/replace-string-action@v1.2
+        with:
+          pattern: ./extension/public/static/package.json
+          string: ${{ github.event.inputs.version }}
+          value: ^(.*?)-
```

Comment:
webstore is picky about versions. You can't use any letters so a version such as 1.0.2-beta.0, for ex, needs to be stripped of everything after the `-`.